stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/message/attachments/internal/UploadAttachmentsWorker.kt

```diff
@@ -41,6 +41,8 @@ internal class UploadAttachmentsWorker(
         channelId: String,
         messageId: String,
     ): Result<Unit> {
+        val message = messageRepository.selectMessage(messageId)
```

Comment:
Hey @leandroBorgesFerreira 

I wonder if we can move this inside the `try-catch` block, just to stay on the safe side.

Reply:
We can't because then we would have to call it again inside the catch block and anything expensive won't run there because it will be cancelled